R/pkg/R/mllib.R

```diff
@@ -41,7 +41,8 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' sqlContext <- sparkRSQL.init(sc)
 #' data(iris)
 #' df <- createDataFrame(sqlContext, iris)
-#' model <- glm(Sepal_Length ~ Sepal_Width, df)
+#' model <- glm(Sepal_Length ~ Sepal_Width, df, family=c("gaussian","binomial"))
```

Comment:
Sorry for the trivial question - I'm new in jira/github world

Should I create a new pull request and change it to : model <- glm(Sepal_Length ~ Sepal_Width, df, family="gaussian") or can I directly commit the changes in the existing one ? 

Thanks,
Narine